README.md

```diff
@@ -1,2 +1,126 @@
+
+[DMTN-093](https://dmtn-093.lsst.io/) presents the technical design of the alert distribution system.
+
+[DMTN-102](https://dmtn-102.lsst.io/) summarizes "Key Numbers" for alerts, estimating numbers, sizing, etc.
+
+[lsst.alert.packet](https://github.com/lsst/alert_packet) is a Python package that contains alert schemas as well as utilities for reading and writing Rubin alert packets.
+
+Community broker authors will be interested in simulating the Kafka alert stream itself.  [alert-stream-simulator](https://github.com/lsst-dm/alert-stream-simulator/) provides the relevant code; the overall design is discussed in 
+[DMTN-149](https://dmtn-149.lsst.io/).
+[alert-stream](https://github.com/lsst-dm/alert_stream) currently contains additional code for converting from the [Confluent Wire Format](https://dmtn-093.lsst.io/#management-and-evolution) used in the Kafka stream.
+
+## Data Sources
+
+### DECam-HiTS
+
+Sample alerts are provided from a subset of the DECam High Cadence Transient Search (HiTS; [Förster et al. 2016](https://ui.adsabs.harvard.edu/abs/2016ApJ...832..155F/abstract)).
+
+We process g-band data from three fields (`Blind15A_26`, `Blind15A_40`, and `Blind15A_42`) from the 2015 campaign and use coadded templates built from the matching fields in the 2014 campaign.  
+In total there are 5020 processed CCD-visits.
+
+Users interested in performing their own reductions can obtain these data [here](https://github.com/lsst/ap_verify_hits2015).
+The `ap_verify` package in the LSST Stack can be used to [process this dataset](https://pipelines.lsst.io/modules/lsst.ap.verify/running.html).
+
+## Limitations and Caveats
+
+The Alert Production pipelines are not yet complete, so not all of the alert contents prescribed by the DPDD are populated in these sample alerts.
+Missing fields will have `null` entries.
+We describe the most significant omissions below.
+
+### Sample
+Each `DIASource` detected at or above 5 sigma in the difference image produces a `DIASource` entry in the PPDB and a corresponding alert packet.
+Because detection SNR is different than the measured SNR of a point-source fit, not all `DIASources` will have `psFlux/psFluxErr > 5`.
+
+The sample is not filtered for artifacts or bad subtractions.
+Through visual inspection we suggest that the current proportion of artifacts to real transients is about 4:1.
+We expect continued improvement in this ratio.
+
+### Formats
+The Avro format, schema structure, and alert contents released here reflect the current baseline, but users should be prepared for continued evolution and potentially breaking changes.
+
+The top-level `AlertID` record is currently not populated.
+
+### `DIASource` Records
+* The detection signal-to-noise (`SNR`) is not populated 
+* Trailed source fits are not implemented
+* No extendedness or spuriousness score is included
+	
+### `DIAObject` Records
+
+* Parallax and proper motion fitting is not performed
+* Timeseries features are not computed
```

Comment:
This is not true right? We do have some Static time series features implemented. Just not the LS columns.

Reply:
We are not propagating them into the alerts themselves, though, correct?  All of the `Lc(Non)Periodic` entries are `null`.

Reply:
Ah, you could write the measurements we currently do into the schema though. Those would produce some of the non-periodic measurements.

Reply:
I'm inclined to save that for the future--both because I want to get this sample out the door and because my next™ project is to work on the timeseries features further.

Reply:
I'll just say "Timeseries features are not _included_"